blog/2022-04-17-starter-calculators.md

```diff
@@ -27,7 +27,7 @@ The following are free-to-use calculators developed by researchers at the Univer
 | ----------------------------------------------------------------------------------------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | [Processor Comparison](https://mecsimcalc.com/app/5558606/processor_comparison)                                   | Hardware Engineering | Compares two processors by utilizing benchmark times for different programs.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-| [Discrete Signals Difference Equations](https://mecsimcalc.com/app/8483456/discrete_signals_difference_equations) | Signal Processing    | An application to determine values of a difference equation. Assumptions are sets the x(n)=δ(n) (the impulse function) and y(n)=0 for n<0. Make sure n is greater than the size of Y coefficients.                                                                                                                                                                                                                                                                                                           |
+| [Discrete Signals Difference Equations](https://mecsimcalc.com/app/8483456/discrete_signals_difference_equations) | Signal Processing    | An application to determine values of a difference equation. Assumptions are sets the x(n)=δ(n) (the impulse function) and y(n)=0 for n{'<'}0. Make sure n is greater than the size of Y coefficients.                                                                                                                                                                                                                                                                                                       |
```

Comment:
not sure if this is supposed to be here, in the app docs for the Discrete Signals Difference Equations app it doesn't contain the "{'<'}" just "n < 0"

Reply:
Yeah when I updated the packages it required me to put < inside the {'<'} in the md file, but it still displays the same on the site.